includes/core/server/request/class-update-account.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * Class file for WCPay\Core\Server\Request\Update_Account.
+ *
+ * @package WooCommerce Payments
+ */
+
+namespace WCPay\Core\Server\Request;
+
+use WCPay\Core\Exceptions\Server\Request\Invalid_Request_Parameter_Exception;
+use WCPay\Core\Server\Request;
+use WC_Payments_API_Client;
+
+/**
+ * Request class for updating account.
+ */
+class Update_Account extends Request {
+	/**
+	 * Returns the request's API.
+	 *
+	 * @return string
+	 */
+	public function get_api(): string {
+		return WC_Payments_API_Client::ACCOUNTS_API;
+	}
+
+	/**
+	 * Returns the request's HTTP method.
+	 */
+	public function get_method(): string {
+		return 'POST';
+	}
+
+	/**
+	 * If true, the request will be signed with the user token rather than blog token.
+	 *
+	 * @return bool
+	 */
+	public function should_use_user_token(): bool {
```

Comment:
Perhaps this function name could be a bit misleading. Either pass an argument and make it default to true or name the function to something like 'use_user_token'

Reply:
This is to override the parent `Request::should_use_user_token` method. If needed, we can open a new PR to fix it for the Request class. But anyway, it should not be addressed in this PR.

Reply:
Yes, you are right. Feel free to ignore my comment.